src/utils/staking.utils.ts

```diff
@@ -0,0 +1,23 @@
+import { FarmToken, FarmTokenStandardsEnum } from 'src/apis/quipuswap-staking/types';
+import { IconNameEnum } from 'src/components/icon/icon-name.enum';
+import { VisibilityEnum } from 'src/enums/visibility.enum';
+import { TokenStandardsEnum } from 'src/token/interfaces/token-metadata.interface';
+import { TokenInterface } from 'src/token/interfaces/token.interface';
+
+export const convertFarmToken = (rawToken: FarmToken): TokenInterface => {
+  const { fa2TokenId, contractAddress, metadata, type } = rawToken;
+  const { name, symbol, decimals, thumbnailUri } = metadata;
+
+  return {
+    balance: '0',
+    visibility: VisibilityEnum.Visible,
+    id: fa2TokenId ?? 0,
+    address: contractAddress === 'tez' ? '' : contractAddress,
```

Comment:
Let's create a constant like `TEZOS_CONTRACT_ADDRESS` in `src/apis/quipuswap-staking/consts.ts` and use wherever necessary